package.json

```diff
@@ -4,8 +4,7 @@
   "description": "A client-side utility class that can call server-side Google Apps Script functions",
   "main": "build/index.js",
   "files": [
-    "build",
-    "src"
+    "build"
```

Comment:
I prefer keeping src in the bundle. It's nice to be able to inspect the src code easily.

Reply:
Add src back in.